packages/dashboard/src/components/dashboard/dashboard.tsx

```diff
@@ -274,6 +278,36 @@ export default function Dashboard(_props: {}): React.ReactElement {
     [dashboardDispatch, appController],
   );
 
+  // temp code with mock data to display status
```

Comment:
These are code that I write to show how it looks like in the dashboard for demonstration purposes. Should we keep these just for display purposes or should we remove it from the dashboard once it is ready to be commited?